src/api/PDFDocument.ts

```diff
@@ -1110,7 +1105,7 @@ export default class PDFDocument {
   async embedPng(png: string | Uint8Array | ArrayBuffer): Promise<PDFImage> {
```

Comment:
Same here

Reply:
You should also be able to remove it from embedSvg. try to look for other impacts.

Reply:
Thx for review. I will check other async function.

Reply:
Thanks for review. Plz check for changes.